arquillian-desktop-video-recorder/pom.xml

```diff
@@ -33,27 +33,20 @@
         </developer>
     </developers>
 
-    <repositories>
-        <repository>
-            <id>xuggle repo</id>
-            <url>http://xuggle.googlecode.com/svn/trunk/repo/share/java</url>
-        </repository>
-    </repositories>
-
     <!-- Properties -->
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    
-        <!-- Xuggler -->
-        <xuggler.version>5.4</xuggler.version>
+
+        <jcodec-javase.version>0.1.9</jcodec-javase.version>
+        <thumbnailator.version>0.4.8</thumbnailator.version>
 
         <!-- Other -->
         <version.maven.deploy.plugin>2.8.1</version.maven.deploy.plugin>
         <version.maven.compiler.plugin>2.3.2</version.maven.compiler.plugin>
 
-        <maven.compiler.target>1.6</maven.compiler.target>
-        <maven.compiler.source>1.6</maven.compiler.source>
-    </properties>    
+        <maven.compiler.target>1.7</maven.compiler.target>
+        <maven.compiler.source>1.7</maven.compiler.source>
```

Comment:
True I read I don't remember where that JCodec requires Java 7 but I have checked now and the requirement is with 6, so I change it right now.